tests/features/python/authentication.py

```diff
@@ -0,0 +1,13 @@
+from behave import *
+
+@given('an anonymous user')
+def step_impl(context):
+    pass
+
+@when('an existing user')
+def step_impl(context):
+    assert True is not False
+
+@then('I sign up with a valid email and password')
+def step_impl(context):
+    assert context.failed is False
```

Comment:
step definitions can be written in other languages - this is good because we can test `supabase-py` here, and `supabase-js` in the test above